components/feature/webnotifications/src/main/java/mozilla/components/feature/webnotifications/WebNotificationFeature.kt

```diff
@@ -66,7 +67,7 @@ class WebNotificationFeature(
         ensureNotificationGroupAndChannelExists()
 
         notificationIdMap[webNotification.tag]?.let {
-            notificationManager?.cancel(it)
+            notificationManager?.cancel(NOTIFICATION_TAG_ID, it)
```

Comment:
Should this be a tag that includes the domain so that we can have multiple concurrent notifications?

Reply:
@pocmo you should still be able to have multiple notifications from what I understand [in the API](https://developer.android.com/reference/android/app/NotificationManager#notify(int,%20android.app.Notification)). The tag + id pair, is what makes the notification unique to app which we were missing. In Fenix, we've been using a still tag with different IDs for multiple tabs received, which seems to work out fine (minus the `Random()` bit 😅 ).

Reply:
As discussed.  Since the tag + id pair will be uniquely identifying each notification.  I'll go ahead and mark this resolved.  Thanks,